pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -22,6 +22,8 @@ type RunRepositoryProvider interface {
 	Create(ctx context.Context, run *models.Run) error
 	// Delete marks existing models.Run entity as deleted.
 	Delete(ctx context.Context, run *models.Run) error
+	// DeleteByExperimentID marks existing models.Run having the given expirimentID as deleted.
+	DeleteByExperimentID(ctx context.Context, experiementID int32) error
```

Comment:
Im not sure that we need a new method. It seems like we already have `Delete(ctx context.Context, run *models.Run) error` and it does it seems like the same what you did. Can you please take a look?

Reply:
hm, what about to have then something like `DeleteByOptions` ?

Reply:
i'd support that when we add another case :-)